woo-tracking-jsdoc/publish.js

```diff
@@ -80,7 +80,12 @@ exports.publish = function ( data ) {
 						/\|/g,
 						'\\|'
 					);
-					mdResult += `\`${ property.name }\` | \`${ type }\` | ${ description }\n`;
+					mdResult += `\`${
+						property.name
+					}\` | \`${ type }\` | ${ description.replace(
+						/\s*\n\s*/,
```

Comment:
💅 
How about adding the `g` flag in case of the `description` has more than one newline char?